src/libstrutil/split.cpp

```diff
@@ -0,0 +1,101 @@
+#include <algorithm>
+#include <fstream>
+#include <iostream>
+#include <sstream>
+
+#include "strutil.hpp"
+
+namespace strutil {
+
+/**
+ * ""           -> ""
+ * ".\t..\t..." -> "" (. is whitespace)
+ * " a  b  c"   -> "a", "b", "c"
+ * "a b c"      -> "a", "b", "c"
+ * "a"          -> "a"
+ */
+static vector<string> splitEmptySep(const string& str) {
+  vector<string> result;
+  std::istringstream iss(str);
+
+  copy(std::istream_iterator<string>(iss), std::istream_iterator<string>(), back_inserter(result));
```

Comment:
```suggestion
  std::istringstream iss(str);
  vector<string> result((std::istream_iterator<string>(iss)), std::istream_iterator<string>());
```

범위 기반 생성자로 나중에 바꾸면 더 좋을 것 같습니다.